scripts/generator/graphql_generator/csharp/SDK/Android/NativeMessage.Android.cs.erb

```diff
@@ -0,0 +1,24 @@
+#if UNITY_ANDROID
+namespace <%= namespace %>.SDK {
+    using System;
+    using System.Collections.Generic;
+    using <%= namespace %>.MiniJSON;
+
+    #if !SHOPIFY_MONO_UNIT_TEST
+    using UnityEngine;
+    #endif
+
+    public partial class NativeMessage {
+        private static AndroidJavaObject MessageCenter;
```

Comment:
I think this will 💥 on Mono tests since `UnityEngine` is wrapped in `!SHOPIFY_MONO_UNIT_TEST`, We should probably define `UNITY_ANDROID`as a symbol in `test.sh`